app/helpers/hostgroup_description_helper.rb

```diff
@@ -0,0 +1,14 @@
+module HostgroupDescriptionHelper
+  UI.register_hostgroup_description do
+    hostgroup_actions_provider :base_hostgroup_actions
+  end
+
+  def base_hostgroup_actions(hostgroup)
+    [
+      { action: display_link_if_authorized(_('Nest'), hash_for_nest_hostgroup_path(id: hostgroup)), priority: 10 },
+      { action: display_link_if_authorized((link_to _('Create Host'), new_host_path(hostgroup_id: hostgroup.id))), priority: 20 },
```

Comment:
this line causes two links to be generated, one for the hostgroups page and one for the host form.
```suggestion
      { action: display_link_if_authorized(_('Create Host'), hash_for_new_host_path(hostgroup_id: hostgroup.id)), priority: 20 },

```

Reply:
Thanks for noticing! Fixing it